addon/components/o-s-s/select.ts

```diff
@@ -23,6 +27,12 @@ export default class OSSSelect extends Component<OSSSelectArgs> {
 
   @tracked displaySelect: boolean = false;
 
+  cleanupDrodpownAutoplacement?: () => void;
```

Comment:
Why added a `?` here?

Reply:
because it doesn't have a value from the start